task.sql

```diff
@@ -68,8 +69,8 @@ CREATE TABLE IF NOT EXISTS likes
     comment_id BIGINT REFERENCES comments (comment_id) ON DELETE CASCADE,
     created_at TIMESTAMP(3) NOT NULL DEFAULT NOW(),
     updated_at TIMESTAMP(3) NOT NULL DEFAULT NOW(),
-    created_by BIGINT REFERENCES users(id) ON DELETE CASCADE,
-    updated_by BIGINT REFERENCES users(id) ON DELETE CASCADE
+    created_by   BIGINT CHECK (created_by > 0),
```

Comment:
I think no need this check

Reply:
Can it stay if it's not wrong? I could write normally, I wrote because it occurred to me @sabuhihuseynov

Reply:
It's foreign key, if created_by will not found the user with given id from users table, it will give you error, that's why we don't need additional checking for foreign keys